<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestPlaneConstructPlaneDialog</class>
 <widget class="QDialog" name="QtTestPlaneConstructPlaneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPlane</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolTestPlane.qrc">
    <normaloff>:/images/ToolTestPlaneR.png</normaloff>:/images/ToolTestPlaneR.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}
QPushButton { background-color: rgb(255, 255, 255);  }
QPushButton { color: rgb(0,0,0); }
QPushButton { font: 48px &quot;Courier New&quot;, courier, monospace }
QPlainTextEdit { font: 12px &quot;Courier New&quot;, courier, monospace }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_1" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Point 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_x1">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_y1">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_z1">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Point 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_x2">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_y2">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_z2">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Point 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_x3">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_y3">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_z3">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Moving point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_x">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_y">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="box_z">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_precision">
     <property name="text">
      <string>Precision (number of digits)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="box_precision">
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="text">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::WidgetWidth</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ToolTestPlane.qrc"/>
 </resources>
 <connections/>
</ui>
